Api/Session.cs

```diff
@@ -243,12 +247,16 @@ public async Task RefreshSession(string refreshToken)
                 { "refresh_token", refreshToken }
             };
 
+            var headerFields = new NameValueCollection()
+            { 
+                { "hardware_id", "unespecified" }
```

Comment:
Is there a reason "unespecified" is misspelled?  Like, does this match against something Ring cares about, or is it just a nonce string to put something in here for hardware_id and thus the spelling doesn't matter?